### ISSUE TITLE
included CE not deleteable -> without hint

### ISSUE THREAD

**Issue by Metis77** [contao/core#8492]
If a CE is included somewhere else, it can not be deleted. The red delete cross is greyed out.

I suggest to add a hint, that this is included somewhere else, and can therefor not be deleted.

Otherwise people, like me, might search for an error solution ...

**Comment by fritzmg** [comment]
Also, if you simply do not remember where you included that CE, it might take you some time until you find the include element.

**Comment by discordier** [comment]
So what do you suggest? A hint will be a static text. Furthermore we can only tell you which references the CE, but a full lookup is costly and will most likely be confusing when displayed.

**Comment by fritzmg** [comment]
I tried something like that with https://github.com/fritzmg/contao-be-include-info
But yeah, it can be confusing, but can also be helpful.

**Comment by Metis77** [comment]
Sure, details would be helpful. 
But may only after clicking an hint. Than detailed info could be shown in a modal.
And with a modal the lookup could be done only if anybody clicks.

**Comment by leofeyer** [comment]
Almost a duplicate of #2953.

**Comment by Mynyx** [comment]
How about adding as first step a tooltip there, that deleting is unavailable because of an include anywhere.
